ibis/client.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, client, sql, **kwargs):
         self.result_wrapper = getattr(dml, 'result_handler', None)
         self.extra_options = kwargs
 
-    def execute(self):
+    def execute(self, **kwargs):
```

Comment:
what are you passing thru here? this is a bit odd to do

Reply:
I need this to allow ipc and gpu_device parameters here https://github.com/ibis-project/ibis/pull/1964/files#diff-2dc1450d8892160d7064e7f6700a2b6fR664

as the other backends don't implement this parameter using `**kwargs` doesn't break anything.

do you have another suggestion?

Reply:
cc @icexelloss didn't you have a PR which does something similar? is this the pattern?

Reply:
rather than doing this, can you simply override execute in the omnisci client? we really don't want to pass thru unamed parameters on a global basis